light-jobs-test/src/main/java/com/hazelcast/jet/tests/lightjobs/LightJobsTest.java

```diff
@@ -0,0 +1,180 @@
+    private static final String SOURCE_JOURNAL_MAP_NAME = "lightJobJournalInputMap";
+    private static final String SOURCE_INCORRECT_MAP_NAME = "lightJobIncorrectMap";
+    private static final String OUTPUT_BATCH_MAP_PREFIX = "lightJobBatchOutputSinkMap";
+    private static final String OUTPUT_STREAM_MAP_PREFIX = "lightJobJournalOutputSinkMap";
+    private static final String OUTPUT_INCORRECT_MAP_NAME = "lightJobIncorrectSinkMap";
+
+    private static final String EXPECTED_FAIL_MESSAGE = "LightJobsTest - incorrectPipeline - expected exception";
+
+    private static final int DEFAULT_ITEM_COUNT = 100;
+    private static final int DEFAULT_LOG_JOB_COUNT_THRESHOLD = 1000;
+    private static final int DEFAULT_SLEEP_BETWEEN_JOBS_MS = 100;
+
+    private IMap<String, String> sourceBatchMap;
+    private IMap<String, String> sourceJournalMap;
+    private IMap<String, String> sourceIncorrectMap;
+
+    private int itemCount;
+    private int sleepAfterJobMillis;
+    private int logJobCountThreshold;
+
+    public static void main(String[] args) throws Exception {
+        new LightJobsTest().run(args);
+    }
+
+    @Override
+    protected void init(HazelcastInstance client) throws Exception {
+        sourceBatchMap = client.getMap(SOURCE_BATCH_MAP_NAME);
+        sourceBatchMap.clear();
+        sourceJournalMap = client.getMap(SOURCE_JOURNAL_MAP_NAME);
+        sourceJournalMap.clear();
+        sourceIncorrectMap = client.getMap(SOURCE_INCORRECT_MAP_NAME);
+        sourceIncorrectMap.clear();
+
+        itemCount = propertyInt("itemCount", DEFAULT_ITEM_COUNT);
+        sleepAfterJobMillis = propertyInt("sleepAfterJobMillis", DEFAULT_SLEEP_BETWEEN_JOBS_MS);
+        logJobCountThreshold = propertyInt("logJobCountThreshold", DEFAULT_LOG_JOB_COUNT_THRESHOLD);
+
+        for (int i = 0; i < itemCount; i++) {
+            String item = Integer.toString(i);
+            sourceBatchMap.put(item, item);
+            sourceJournalMap.put(item, item);
+        }
+        sourceIncorrectMap.put("ignore", "ignore");
+    }
+
+    @Override
+    protected void test(HazelcastInstance client, String name) {
+        long begin = System.currentTimeMillis();
+        long jobCount = 0;
+        while (System.currentTimeMillis() - begin < durationInMillis) {
+            runBatchJob(client, jobCount);
+            sleepMillis(sleepAfterJobMillis);
+            runStreamJob(client, jobCount);
+            sleepMillis(sleepAfterJobMillis);
+            runIncorrectJob(client);
+            sleepMillis(sleepAfterJobMillis);
+
+            if (jobCount % logJobCountThreshold == 0) {
+                logger.info("Job count: " + jobCount);
+            }
+            jobCount++;
+        }
+        logger.info("Final job count: " + jobCount);
+    }
+
+    private void runBatchJob(HazelcastInstance client, long jobCount) {
+        String outputName = OUTPUT_BATCH_MAP_PREFIX + jobCount;
```

Comment:
we can use IList instead of IMap for the sink. IList is not a distributed data-structure therefore it is not that costly to create/destroy. Since we add the jobCount to the name, it'll be a different partition for each job.